eng/sdk/sync_sdk.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License.
+
+import os
+import sys
+import logging
+import argparse
+import subprocess
+import glob
+import json
+from typing import List
+
+sdk_root: str
+
+skip_artifacts: List[str] = ['azure-ai-anomalydetector']
```

Comment:
Skip azure-ai-anomalydetector for now.

Its commit id in SDK repo is too old, that `tsp compile` will fail.